buildshiprun/handler.go

```diff
@@ -67,6 +67,7 @@ func Handle(req []byte) string {
 				"Git-Owner":      owner,
 				"Git-Repo":       repo,
 				"Git-DeployTime": strconv.FormatInt(time.Now().Unix(), 10), //Unix Epoch string
+				"Git-SHA":        getGitSHA(imageName),
```

Comment:
I was hoping we would pass it in from the previous method.. are we able to do that?

Reply:
oh yeah .. we can do that too. I think that would be a better way to do it.